src/main/java/com/google/api/generator/Main.java

```diff
@@ -20,15 +20,13 @@
 import com.google.api.ResourceProto;
 import com.google.api.generator.gapic.Generator;
 import com.google.longrunning.OperationsProto;
-import com.google.protobuf.Descriptors.DescriptorValidationException;
 import com.google.protobuf.ExtensionRegistry;
 import com.google.protobuf.compiler.PluginProtos.CodeGeneratorRequest;
 import com.google.protobuf.compiler.PluginProtos.CodeGeneratorResponse;
 import java.io.IOException;
 
 public class Main {
-  public static void main(String[] args)
-      throws IOException, InterruptedException, DescriptorValidationException {
+  public static void main(String[] args) throws IOException {
```

Comment:
@miraleung (from https://github.com/googleapis/gapic-generator-java/pull/767): Could we move this into a separate file? Having the main classes all call something like `Registry.registerAllExtensions` has more consistent semantics than `Main.registerAllExtensions`.

Up to you whether to put this in a subpackage or keep it here.

Reply:
Ok